entrypoint.sh

```diff
@@ -22,9 +22,20 @@ if [[ "${HELM_DEBUG}" == "true" ]]; then
 	DEBUG_ARGS=" --debug"
 fi
 
-if [[ "${OPERATION}" == "deploy" ]]; then
+# approach 1: --state-values-set flag. This doesn't seem to propagate from helmfile.yaml to the imported release helmfile
+if [[ -n "$HELM_VALUES" ]]; then
+  HELM_VALUES_FLAG="--state-values-set ${HELM_VALUES}"
+fi
+
+# apprach 2: contents of a file to include in helmfile.yaml
+if [[ -n "$HELM_VALUES_YAML" ]]; then
+  echo -e "Using extra values:\n${HELM_VALUES_YAML}"
+  export HELM_VALUES_FILE="/tmp/extra_helm_values.yml"
+  echo "$HELM_VALUES_YAML" > "$HELM_VALUES_FILE"
+fi
 
-	OPERATION_COMMAND="helmfile --namespace ${NAMESPACE} --environment ${ENVIRONMENT} --file /deploy/helmfile.yaml $DEBUG_ARGS apply"
+if [[ "${OPERATION}" == "deploy" ]]; then
+	OPERATION_COMMAND="helmfile ${HELM_VALUES_FLAG}  --namespace ${NAMESPACE} --environment ${ENVIRONMENT} --file /deploy/helmfile.yaml $DEBUG_ARGS apply"
```

Comment:
This change is only needed with the `--state-values-set` approach, and goes away if we focus on the yaml file approach.